dist/index.html

```diff
@@ -8,17 +8,23 @@
     <link rel="stylesheet" href="./css/style.css">
 </head>
 <body>
-    <div class="container">
+    <div class="container desk-container">
         <header class="menu">
             <a href="#" class="logo">Crystal logo</a>
             <nav class="burger">
               <span></span>
               <span></span>
               <span></span>
             </nav>
+            <nav class="desktop-menu">
+              <a href="#">Portfolio</a>
```

Comment:
- [ ] please note that the headline section of the desktop version is not centralized. It is aligned to the left. Kindly fix yours so it follows the original design.
- [ ] kindly follow the font sizes and weight as well

yours
![Screenshot (1049)](https://user-images.githubusercontent.com/65298048/236388558-7007ea9a-fa77-41f0-ab94-d5ba35a8e1ee.png)
figma
![Screenshot (1050)](https://user-images.githubusercontent.com/65298048/236388627-4d5aaf53-3337-457f-80b8-2218ba83b938.png)